docs/data-sources/caller_identity.md

```diff
@@ -0,0 +1,28 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "awsutils_caller_identity Data Source - terraform-provider-awsutils"
+subcategory: ""
+description: |-
+  Use this data source to get the access to the effective
+          Account ID, User ID, ARN and EKS Role ARN in which Terraform is authorized.
+---
+
+# awsutils_caller_identity (Data Source)
+
+Use this data source to get the access to the effective
+		Account ID, User ID, ARN and EKS Role ARN in which Terraform is authorized.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Read-Only
+
+- `account_id` (String)
+- `arn` (String)
+- `eks_role_arn` (String)
+- `id` (String) The ID of this resource.
+- `user_id` (String)
```

Comment:
```suggestion
## Attributes Reference

* `account_id` - AWS Account ID number of the account that owns or contains the calling entity.
* `arn` - ARN associated with the calling entity.
* `eks_role_arn` - If the calling identity is an assumed role, this is the transformation of that assumed role ARN into something suitable for use in configuring EKS access. See [Add IAM principals to your Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html#aws-auth-users). Otherwise, `null`.
* `id` - Same as `account_id`.
* `user_id` - Unique identifier of the calling entity.
```

Reply:
Done. Note that the docs are auto-generated from the schema descriptions, so I updated the source rather than the resulting docs. See [322dcaa](https://github.com/cloudposse/terraform-provider-awsutils/pull/39/commits/322dcaa0d60213e10b5caad5651cf398cff8192d)